core/src/main/java/io/github/thibaultbee/streampack/internal/encoders/MediaCodecEncoder.kt

```diff
@@ -150,6 +151,11 @@ abstract class MediaCodecEncoder<T : Config>(
             Handler(handlerThread.looper)
         }
     }
+    private fun releaseHandler() {
```

Comment:
currently I believe that the MCE leaks a thread every time the user hits start/stop? This code is not probably optimal. Maybe better to reuse the same handler on subsequent starts